### PR TITLE
updated_vad

### PR DESCRIPTION
**Pull request by ipa-nhg**

### CODE REVIEW

**Comment by batikim09** [comment]
I merged the recent push into my repository and tried to make a new pull request.
It says|:
 "Able to merge. These branches can be automatically merged."
Actually merge will be possibly done by you, right? 

Is there any more thing I should do for merge?  
I'm not familiar with this so questions keep coming :(
If you give me more details, It will be a great help. Thank you in advance!

File: squirrel_vad_msgs/package.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  <!-- Use build_depend for packages you need at compile time: -->
+  <!--   <build_depend>message_generation</build_depend> -->
+  <!-- Use buildtool_depend for build tool packages: -->
+  <!--   <buildtool_depend>catkin</buildtool_depend> -->
+  <!-- Use run_depend for packages you need at runtime: -->
+  <!--   <run_depend>message_runtime</run_depend> -->
+  <!-- Use test_depend for packages you need only for testing: -->
+  <!--   <test_depend>gtest</test_depend> -->
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>roscpp</build_depend>
```
Comment (NadiaHG) [inline squirrel_vad_msgs/package.xml:43]: rosccp ist not needed to generate a msg

File: squirrel_vad_msgs/package.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+<build_depend>message_generation</build_depend>
+  <build_depend>rospy</build_depend>
```
Comment (NadiaHG) [inline squirrel_vad_msgs/package.xml:45]: rospy ist not needed to generate a msg

**Comment by ipa-nhg** [comment]
Now it is ok, I have to trigger new jobs to check that your packages fit with the rest of the squirrel repositories. 

Bajo wrote a small guide about how to use github:
https://github.com/squirrel-project/squirrel_recommender#Git-workflow 

But basically you have to commit your changes to your fork/branch and open a pull request to squirrel-project. If a merge is required (because a merge conflict) I will ask you to update your version.

**Comment by batikim09** [comment]
OK, thanks. That guide looks helpful.

**Pull request merged**